### PR TITLE
Add dtype arg to ES in examples

### PR DESCRIPTION
**Pull request by banasraf**
Signed-off-by: Rafal <Banas.Rafal97@gmail.com>

## Description
- [ ] **Bug fix** (*non-breaking change which fixes an issue*)
- [ ] **New feature** (*non-breaking change which adds functionality*)
- [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
- [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
- [x] **Other** (*e.g. Documentation, Tests, Configuration*)

### What happened in this PR

I added the dtype argument to all of occurrences of ES in our examples.

#### Additional information
- Affected modules and functionalities:
  Notebooks

- Key points relevant for the review:
  Please check for any unwanted changes in notebook. 

### Tests
- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**:  DALI-2535

### CODE REVIEW

**Comment by banasraf** [comment]
!build

**Comment by banasraf** [comment]
!build

**Comment by banasraf** [comment]
!build

**Comment by banasraf** [comment]
!build

**Review by prak-nv — APPROVED** [review]
LGTM

File: docs/examples/frameworks/mxnet/mxnet-external_input.ipynb [hunk 1]
```diff
@@ -103,7 +103,7 @@
     "def ExternalSourcePipeline(batch_size, num_threads, device_id, external_data):\n",
     "    pipe = Pipeline(batch_size, num_threads, device_id)\n",
     "    with pipe:\n",
-    "        jpegs, labels = fn.external_source(source=external_data, num_outputs=2)\n",
+    "        jpegs, labels = fn.external_source(source=external_data, num_outputs=2, dtype=dali.types.UINT8)\n",
```
Comment (jantonguirao) [inline docs/examples/frameworks/mxnet/mxnet-external_input.ipynb:106]: any reason why not to import `types` here as in the other notebooks?

**Review by jantonguirao — APPROVED** [review]

**Pull request merged**